### ISSUE TITLE
Reference title or url variables multiple times

### ISSUE THREAD

**Issue by gascarrier** [zaki-yama/copy-title-and-url-as-markdown#348]
Taking google.com for example, the pattern `${title} - [${title}](${url})`, will becomes, `Google - [${title}](https://www.google.com/)`

Can you fix it pls?

**Comment by zaki-yama** [comment]
@gascarrier 
Thank you for reporting!
It seems a bug, so I'll investigate it later.

**Comment by zaki-yama** [comment]
@gascarrier 
I released the latest version 2.0.1, the issue was fixed in the version.
Please check and feel free to ask me if you still have problems.

Thanks.

**Comment by gascarrier** [comment]
> @gascarrier I released the latest version 2.0.1, the issue was fixed in the version. Please check and feel free to ask me if you still have problems.
> 
> Thanks.

Thank you, it's been fixed! awesome!!!

This is the result that I use your extension: 
```
Reference title or url variables multiple times · Issue #348 · zaki-yama/copy-title-and-url-as-markdown - [https://github.com/zaki-yama/copy-title-and-url-as-markdown/issues/348](https://github.com/zaki-yama/copy-title-and-url-as-markdown/issues/348)
```
